src/types/command/CommandEnum.php

```diff
@@ -35,4 +36,8 @@ public function getName() : string{
 	public function getValues() : array{
 		return $this->enumValues;
 	}
+
+	public function isSoft() : bool{
```

Comment:
Please document this. While we use the same naming as vanilla, "soft" is not a great description of what the value actually signifies (it's a dynamic enum that can be updated during the game session).

Reply:
Done